tests/installation/typescript-types.spec.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { test } from './npmTest';
+
+test('typescript types should work', async ({ exec, tsc, writeFiles }) => {
+  const env = { PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: '1' };
+  await exec('npm i -D @types/node@14.18.9');
```

Comment:
Originally, this test checks against typescript 3.7.5 & thus installs a very particular version of node types. I don't think this is important though, and with this PR we'll switch typechecking to ts 3.8 altogether.

I'll need someone to confirm typechecking everything against ts3.8 is good!

```suggestion
```